ui/components/MesherySettings.js

```diff
@@ -351,9 +373,13 @@ class MesherySettings extends React.Component {
     ]
 
     const options = {
-      filter : false,
+      filter : true,
```

Comment:
Would you please turn off the filters? 
Considering UI doesn't implement filters for these data-tables.

![Screenshot 2023-05-01 at 7 58 12 PM](https://user-images.githubusercontent.com/73700530/235467744-4e7328c1-5e53-46d6-afca-522e70dec696.png)

Reply:
Yep, I have turned off the filters

Reply:
Thanks